packages/feedback-spa/src/app/app.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component } from '@angular/core';
 
 @Component({
+  // tslint:disable-next-line: component-selector
```

Comment:
@anjneeksharma This isn't quite recommended. Check out this StackOverflow answer to understand how to do this more gracefully: https://stackoverflow.com/questions/41248142/angular2-how-to-change-the-default-prefix-of-component-to-stop-tslint-warnings